test-two/src/helpers/http.js

```diff
@@ -1,18 +1,55 @@
 import fetch from "node-fetch";
 
-export const get = async url => {
+export const request = async (url, requestInit = {}) => {
   const response = await fetch(url, {
-    method: "GET",
-    headers: {
-      "User-Agent": "request"
-    }
+    ...requestInit,
   });
 
   const json = await response.json();
 
   if (response.status === 200) {
     return json;
   } else {
-    throw Error(json.Message);
+    const error = json.message || json.Message || json;
+    throw Error(error);
   }
+}
+
+export const get = async (url, requestInit = {}) => {
+  return await request(url, {
+    ...requestInit,
+    method: "GET"
+  });
 };
+
+export const post = async (url, requestInit = {}) => {
+  return await request(url, {
+    ...requestInit,
+    method: "POST"
+  });
+}
+
+export const joinPaginatedRequests = async (
+  total,
+  pageSize,
+  getter,
+) => {
+  let i = 0;
```

Comment:
is this being used?

Reply:
no... I did another PR with typescript and eslint, and I captured there... sorry for that